src/sqlfluff/core/rules/analysis/select.py

```diff
@@ -134,7 +134,7 @@ def _has_value_table_function(table_expr, dialect):
         # Other rules can increase whitespace in the function name, so use strip to
         # remove
         # See: https://github.com/sqlfluff/sqlfluff/issues/1304
-        if function_name.raw.lower().strip() in dialect.sets("value_table_functions"):
+        if function_name.raw.upper().strip() in dialect.sets("value_table_functions"):
```

Comment:
Why did this need to change? Did we have dialects using both upper and lower-case to specify value table functions?

Reply:
I noted above in the "Are there any other side effect..." section:

> I uppercased value table values in common with other syntax

Basically, I personally just found it weird that other functions and keywords were in caps,  but these were in lowercase.

I considered making it case insensitive but there's a (very slight) performance impact as have to create a new set, and even though the sets are small it's not worth it IMHO.